website_sales_order_and_reservation_process/__openerp__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Website sales order and reservation process',
+    'version': '8.0.1.0.1',
+    'category': 'web',
+    'depends': [
+        'website_sale',
+        'website_sale_collapse_categories',
```

Comment:
@elicoidal some modification is base on this two dependencies. It's a special module for the jingma project. 
@Rona111 I thinks you need some extra time to remove these 2 dependencies, and modify the module for just depends on website_sale.